src/LaunchDarkly.Client/InMemoryFeatureStore.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+
+namespace LaunchDarkly.Client
+{
+    public class InMemoryFeatureStore : IFeatureStore
+    {
+        private static readonly int RwLockMaxWaitMillis = 1000;
```

Comment:
We might not want to cap this wait time at all.. It seems very unlikely it will ever get hit.